websites/documentation/gatsby-config.js

```diff
@@ -22,6 +22,13 @@ module.exports = {
       additionalPrismLanguages: ['java', 'scala', 'csharp', 'swift', 'php'],
       addOns: [
         '@commercetools-docs/gatsby-theme-code-examples',
+        {
+          resolve: '@commercetools-docs/gatsby-theme-learning',
+          options: {
+            auth0Domain: 'auth.id.commercetools.com',
+            learnApiBaseUrl: 'https://api.learn.commercetools.com',
+          },
+        },
```

Comment:
I don't think it's a smart move to have anybody be able to log into our SSO in the kit documentation website.   It would be very misleading.   I can live with not having a live preview in that situation.

Reply:
@nkuehn It was added already to the smoke's test site, I just added it here as well .. But I understand your concern ... should I remove it at all ? and just keep the documentation for it ?

Reply:
Smoke test has testing purposes and should normally be excluded from google.  OK to keep it there if the content makes clear it's a testing site. 

Documentation is linked from the repo homepage and we may even make it indexed if it's at least a little complete.